README.md

```diff
@@ -623,6 +624,11 @@ Following an example:
         }
 ```
 
+## Delete user data
+Utilize the following method if an end-user requests to have their data deleted:
+```
```

Comment:
We could add the snippets for java and kotlin
```kotlin
    import com.sourcepoint.cmplibrary.util.clearAllData

    //...
    clearAllData(context)
    //...
```

```java
    import com.sourcepoint.cmplibrary.util.SpUtils;

    //...
    SpUtils.clearAllData(context)
    //...
```

Reply:
Done